cfme/common/provider.py

```diff
@@ -75,6 +88,7 @@ def __init__(self, **kwargs):
             self.candu = kwargs.get('candu')
             self.domain = kwargs.get('domain')
             self.ssh = kwargs.get('ssh')
+            self.token = kwargs.get('token')
```

Comment:
Absolutely. I will nod at and agree with every comment here as I want it merged.